Bottom: 31beeab5b3a0a294a66f70f8658c03b8596e91f1
Top:    6a68b77fab118a10cfd25f7935e2dcc5968c4d87
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-08 10:42:18 +0800

Refresh of dev-reproducer

---

diff --git a/net/core/dev.c b/net/core/dev.c
index 82b5478..08c5a85 100644
--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -7674,8 +7674,10 @@ int register_netdevice(struct net_device *dev)
 	else if (__dev_get_by_index(net, dev->ifindex))
 		goto err_uninit;
 
-	if (!strcmp(dev->name, "tap0"))
+	if (!strcmp(dev->name, "tap0")) {
+		printk("trigger!\n");
 		goto err_uninit;
+	}
 
 	/* Transfer changeable features to wanted_features and enable
 	 * software offloads (GSO and GRO).
